files/settings-default.cfg

```diff
@@ -1,244 +1,709 @@
+# disabled.
+gamma = 1.00
 
-inventory companion x = 0
-inventory companion y = 0.4275
-inventory companion w = 0.6225
-inventory companion h = 0.5725
+[Water]
 
-container x = 0.25
-container y = 0
-container w = 0.75
-container h = 0.375
+# The water settings can be tested experimentally in the Water tab of
+# the Video panel in the Options menu.  Changes there will be saved to
+# these settings.
+
+# This boolean setting enables the refraction rendering feature of the
+# water shader.  Refraction causes deep water to be more opaque and
+# objects seen through the plane of the water to have a wavy
+# appearance.  Enabling this feature results in better visuals, and a
+# marginally lower framerate depending on your graphics hardware.  The
+# "shader" setting in the Water section must be enabled for this
+# setting to have any effect.
+refraction = false
 
-companion x = 0.25
-companion y = 0
-companion w = 0.75
-companion h = 0.375
+# Refracted texture size.  In the Video panel of the options menu, the
+# choices are Low (512), Medium (1024) and High (2048).  This setting
+# determines the resolution of the textures used for rendering objects
+# on the other wide of the plane of water (which have a wavy
+# appearance caused the by the refraction).  Higher values produces
+# better visuals and result in a marginally lower framerate depending
+# on your graphics hardware.  The "refraction" setting in the "Water"
+# section must be enabled for this setting to have any effect.
+rtt size = 512
 
-map x = 0.625
-map y = 0
-map w = 0.375
-map h = 0.5725
+# This boolean setting enables or disables the water shader, which
+# results in much more realistic looking water surfaces, including
+# shadows of reflected objects.
+shader = false
 
-barter x = 0.25
-barter y = 0
-barter w = 0.75
-barter h = 0.375
+[Windows]
+
+# Each window in the GUI mode remembers it's previous location.  Each
+# setting is a floating point number representing a fraction of the
+# "resolution x" or "resolution y" setting in the Video section.  The
+# X and Y values locate the top left corner, while the W value
+# determines the width of the window and the H value determines the
+# height of the window.
 
+# The alchemy window, for crafting potions.  Activated by dragging an
+# alchemy tool on to the rag doll.  Unlike most other windows, this
+# window hides all other windows when opened.
+alchemy h = 0.5
+alchemy w = 0.5
 alchemy x = 0.25
 alchemy y = 0.25
-alchemy w = 0.5
-alchemy h = 0.5
 
-stats x = 0
-stats y = 0
-stats w = 0.375
-stats h = 0.4275
+# The NPC bartering window, displaying goods owned by the shopkeeper
+# while bartering.  Activated by clicking on the "Barter" choice in
+# the dialog window for an NPC.
+barter h = 0.375
+barter w = 0.75
+barter x = 0.25
+barter y = 0
 
-spells x = 0.625
-spells y = 0.5725
-spells w = 0.375
-spells h = 0.4275
+# Unused?
+companion h = 0.375
+companion w = 0.75
+companion x = 0.25
+companion y = 0
```

Comment:
Used by the companion UI, e.g. [Calvus Horatius](http://www.uesp.net/wiki/Tribunal:Calvus_Horatius) in the Tribunal expansion.